bin/ants

```diff
@@ -156,6 +156,11 @@ def run_ansible(args):
     if args.skip_tags:
         cmd.append('--skip-tags')
         cmd.append(args.skip_tags)
+    if configer.read_config('logstash'):
+        cfg_logstash = configer.read_config('logstash')
+        os.environ['ANSIBLE_CALLBACK_WHITELIST'] = CFG['ansible_callback_whitelist']
```

Comment:
I'm not sure whether we should change os.environ directly or copy it first: https://stackoverflow.com/questions/2231227/python-subprocess-popen-with-a-modified-environment

What do you think?

Reply:
I would not copy it but add our attributes to it if they are not already in the variable. I will do this tomorrow, too.